controllers/categoriesController.js

```diff
@@ -0,0 +1,103 @@
+const BaseController = require("./baseController");
+
+class CategoriesController extends BaseController {
+  constructor(model, db, bookModel) {
+    super(model);
+    this.post = db.post;
+    this.like = db.like;
+    this.comment = db.comment;
+    this.bookModel = bookModel;
+    this.sequelize = db.sequelize;
+  }
+
+  async getCategories(req, res) {
+    try {
+      const categories = await this.model.findAll({
+        attributes: ["id", "name"],
+      });
+      return res.json(categories);
+    } catch (err) {
+      return res.status(400).json({ error: true, msg: err });
+    }
+  }
+
+  async getCategory(req, res) {
+    const { category } = req.params;
+    try {
+      const selectedCategory = await this.model.findOne({
+        where: {
+          name: category,
+        },
+      });
+      const selectedBooks = await selectedCategory.getBooks();
+      return res.json(selectedBooks);
+    } catch (err) {
+      return res.status(400).json({ error: true, msg: err });
+    }
+  }
```

Comment:
I think it should be in the book route because you want to get books by the category